clang/test/CodeGenSYCL/max-concurrency.cpp

```diff
@@ -0,0 +1,81 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -disable-llvm-passes -triple spir64-unknown-unknown-sycldevice -sycl-std=2020 -emit-llvm -o - %s | FileCheck %s
+
+#include "sycl.hpp"
+
+// CHECK: br label %for.cond,   !llvm.loop ![[MD_MC:[0-9]+]]
```

Comment:
It'd be really handy to add some `CHECK-LABEL` lines to give FileCheck some more context as to where to search for these matches. @erichkeane may have some best practices on this.

Reply:
@erichkeane What do you suggest adding as additional checks?

Reply:
I tend to do a CHECK-LABEL at the beginning of each function (for the function name) and at some point at the beginning of the metadata to make debugging when these stop working easier.